drivers/sensor/bme680/Kconfig

```diff
@@ -0,0 +1,157 @@
+# Kconfig.bme680 - BME680 temperature, pressure, humidity and gas sensor configuration options
+
+#
+# Copyright (c) 2016, 2017 Intel Corporation
+# Copyright (c) 2017 IpTronix S.r.l.
+# Copyright (c) 2018 Bosch Sensortec GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+menuconfig BME680
+	bool "BME680 sensor"
+	depends on I2C
+	default	n
```

Comment:
This is redundant and should be removed (like it was done in [this commit](https://github.com/zephyrproject-rtos/zephyr/commit/19d78035460638054bf6c4c182b32774f48d6d3a) for other sensors).

Reply:
ok